flowzone.yml

```diff
@@ -4090,18 +4090,20 @@ jobs:
         run: |
           template_base_directory="$(dirname '${{ steps.shared.outputs.template_file }}')"
 
-          python_lambdas="$(cat '${{ steps.shared.outputs.template_file }}' \
+          python_lambdas="$(cat <'${{ steps.shared.outputs.template_file }}' \
```

Comment:
What is this change helping with?

Reply:
There's some fascinating reading [here](https://unix.stackexchange.com/questions/258931/difference-between-cat-and-cat), especially in the comments and pointing to more SO posts.

I am standardising on `cat <input` and `cat >output`, so whenever I see a plain `cat` I change it.